cdap-app-fabric/src/main/java/io/cdap/cdap/internal/metadata/MetadataConsumerSubscriberService.java

```diff
@@ -265,35 +265,34 @@ public void processMessage(MetadataMessage message, StructuredTableContext conte
         run = getProgramRunForConsumer(programRunId, startTimeMs, endTimeMs);
         info = getLineageInfoForConsumer(fieldLineageInfo, startTimeMs, endTimeMs);
       } catch (IllegalArgumentException e) {
-        LOG.warn(
-            "Error while processing field-lineage information received from TMS. Ignoring : {}",
-            message, e);
+        LOG.warn("Error while processing field-lineage information received from TMS. Ignoring : {}", message, e);
         return;
       }
       this.consumers.forEach((key, consumer) -> {
         try {
           DefaultMetadataConsumerContext metadataConsumerContext =
-              new DefaultMetadataConsumerContext(cConf, consumer.getName());
+            new DefaultMetadataConsumerContext(cConf, consumer.getName(), this.metricsCollectionService,
+                                               run.getNamespace());
+          MetadataConsumerMetrics metrics = metadataConsumerContext.getMetrics(Collections.emptyMap());
+          metrics.increment("metadata.consumer.calls.attempted", 1);
```

Comment:
Just to confirm , the intention is to capture attempted calls regardless of whether they are successful ? And no tags required  for the metric in this context?

Reply:
Yes that's right